documents/operating.md

```diff
@@ -270,6 +274,10 @@ Overlords have three months to cough up the extra dough.
 
 Failure to contribute in a timely manner shall result in revocation of Corporate Overlord status.
 
+A former Corporate Overlord, returning to BIFFUD as described in Section 2 ->
+"**Revenge of the Corporate Overlords**," may have the capital contribution
```

Comment:
I'd be open to this waiving just being automatic -- in fact, it probably should be!

We do need to make explicit what the equity stake of the CO is (both after leaving and coming back).

I'll need to research how this actually is supposed to work.